content/docs/setup/kubernetes/multicluster/gateways/index.md

```diff
@@ -52,40 +52,24 @@ on **each** Kubernetes cluster.
     by a common root CA.
     {{< /tip >}}
 
-1. Generate a multicluster-gateways Istio configuration file using `helm`:
-
-    {{< warning >}}
-    If you're not sure if your `helm` dependencies are up to date, update them using the
-    command shown in [Helm installation steps](/docs/setup/kubernetes/install/helm/#installation-steps)
-    before running the following command.
-    {{< /warning >}}
-
-    {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
-        -f @install/kubernetes/helm/istio/values-istio-multicluster-gateways.yaml@ > $HOME/istio.yaml
-    {{< /text >}}
-
-    For further details and customization options, refer to the
-    [Installation with Helm](/docs/setup/kubernetes/install/helm/) instructions.
-
 1. Run the following commands in **every cluster** to deploy an identical Istio control plane
    configuration in all of them.
 
     * Create a Kubernetes secret for your generated CA certificates using a command similar to the following. See [Certificate Authority (CA) certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key) for more details.
 
         {{< text bash >}}
-        $ kubectl create namespace istio-system
+        $ kubectl create namespace istio-system --dry-run -o yaml | kubectl apply -f -
```

Comment:
@vadimeisenbergibm Can you help me understand why the `--dry-run` flag is needed?

Reply:
This is the same issue as in https://github.com/istio/istio.io/pull/3479, to create the `istio-system` namespace if it does not exist (if we did not create it in some step before).

We should decide on https://github.com/istio/istio.io/pull/3479, and apply the same decision here.